.env.schema

```
@@ -0,0 +1,95 @@
+# ----------
+
+# The app mode controls the varlock validation:
+# on dev all env variables get validated
+# on build only the for build required variables get validated (good for docker build)
+# on prod only the runtime required variables get validated (good for docker run)
+# @type=enum(dev, ci, build, prod)
+APP_MODE=dev
+
+# @type=enum(debug, info, warn, error)
+LOG_LEVEL=ifs(
+    eq(APP_MODE, dev), debug,
```

dotenv-linter (4.0.0)

[warning] [IncorrectDelimiter] The eq(APP_MODE, dev), debug, key has incorrect delimiter

(IncorrectDelimiter)